tests/integration/session_endianness/messages.rflx

```diff
@@ -0,0 +1,16 @@
+package Messages is
+
+   type Integer is mod 2 ** 32;
+
+   type Msg is
+      message
+         A : Integer;
+      end message;
+
+   type Msg_LE is
+      message
+         B : Integer;
+      end message
+         with Byte_Order => Low_Order_First;
+
+end Messages;
```

Comment:
I was assuming the above are the "odd" literal values, and I used the simpler ones in the test. Regardless, the test now also uses an enum type in addition to the integer type.

Reply:
The specification was just meant as an example. There is no need to use exactly this type in the test. Actually, I think it is quite strange that the test uses now a "Measurement_Hash_Algo" type instead of a generic type name like used for the integer.

Reply:
I see no harm in the test using application-inspired names, but in any case I changed the test to use generic names.